packages/next/src/migrations/update-11-5-0/remove-tsconfig-app-11-5-0.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  getProjects,
+  Tree,
+  readJson,
+  updateJson,
+  formatFiles,
+} from '@nrwl/devkit';
+
+export async function removeTsconfigApp(host: Tree) {
+  const projects = getProjects(host);
+
+  projects.forEach((project) => {
+    const tsconfigAppPath = `${project.root}/tsconfig.app.json`;
+    const tsconfigPath = `${project.root}/tsconfig.json`;
+
+    if (project.targets?.build?.executor !== '@nrwl/next:build') return;
+
+    if (!host.exists(tsconfigAppPath)) return;
+
+    const tsconfigAppContents = readJson(host, tsconfigAppPath);
+
+    updateJson(host, tsconfigPath, (json) => {
+      json.include = tsconfigAppContents.include;
+      if (json.references) {
```

Comment:
We can delete the references entry for now. Bring it back to the original version here: https://github.com/nrwl/nx/commit/4f545a131679970a2d126081bd40722a41a2fe21#diff-be07232c875b78b4904c36aae3d7523c68d5fe32bba1293f300233b903b6ad4d